docs/eigenda/overview.md

```diff
@@ -5,8 +5,8 @@ title: EigenDA Overview
 
 EigenDA is a data availability store made by
 [EigenLabs](https://www.eigenlayer.xyz/about) and built on top of
-[EigenLayer](../eigenlayer/overview/README.md), live on Goerli
-[testnet](https://blobs-goerli.eigenda.xyz/) and launching on mainnet in early
+[EigenLayer](../eigenlayer/overview/README.md), currently in transition from the
+Goerli testnet to the Holesky testnet and launching on mainnet in early
 Q2 2024.
 
 EigenDA stores rollup transactions until their computed state is
```

Comment:
This sentence doesn't make any sense and should be removed or replaced with a coherent and complete statement about how EigenDA stores transactions submitted by rollups.

Reply:
Disagree, it makes perfect sense. What part do you not understand or disagree with?

Reply:
I got it now - the sentence looked incomplete in this PR review panel.

I can see the entire segment now - All good @teddyknox